tests/conftest.py

```diff
@@ -315,6 +315,7 @@ def get_app_config(database, nondefaults=None):
         "opensearch.url": "https://localhost/warehouse",
         "files.backend": "warehouse.packaging.services.LocalFileStorage",
         "archive_files.backend": "warehouse.packaging.services.LocalArchiveFileStorage",
+        "archive_files.path": "/var/opt/warehouse/packages-archive/",
```

Comment:
Flagging: I cargo-culted this from `dev/environment`. These changes need this setting as well, since the `IntegrityService` constructs an `IFileStorage` with `name="archive"` under the hood, which translates to `LocalArchiveFileStorage` in the local environment. That in turn requires `archive_files.path`, per:

```py
@implementer(IFileStorage)
class LocalFileStorage(GenericLocalBlobStorage):
    @classmethod
    def create_service(cls, context, request):
        return cls(request.registry.settings["files.path"])
```

Reply:
No longer relevant.